### ISSUE TITLE
Prisma migrate command could be outdated

### ISSUE THREAD

**Issue by catherineluse** [prisma/web#293]
On this page 
https://github.com/prisma/prisma2-docs/blob/master/content/03-reference/01-tools-and-interfaces/03-prisma-migrate.mdx

I ran the prisma migrate commands and got:

```
$ prisma migrate save --experimental
prisma migrate up --experimental
 ▸    migrate save is not a prisma command.
 ▸    Perhaps you meant generate
 ▸    Run prisma help for a list of available commands.

Get in touch if you need help: https://slack.prisma.io
To get more detailed output, run $ export DEBUG="*"
$ prisma migrate up --experimental
 ▸    migrate up is not a prisma command.
 ▸    Perhaps you meant generate
 ▸    Run prisma help for a list of available commands.
```

I think the commands might need to get updated in the doc.

**Comment by janpio** [comment]
This looks a bit like you are running the old `prisma` CLI of Prisma 1 instead of `@prisma/cli` (which also has the executable name `prisma`). Not very clear, we know.

Did you follow the installation instructions?

**Comment by catherineluse** [comment]
I was originally following a course that used Prisma 1, but then I decided to start over with Prisma 2 following the installation docs for migration. https://www.prisma.io/docs/getting-started/setup-prisma/start-from-scratch-prisma-migrate-typescript-postgres.

I thought this command in that guide was going to get the right Prisma CLI version:

```
npm init -y
npm install @prisma/cli typescript ts-node @types/node --save-dev
```

But my Prisma CLI version still was still v1 because I had installed it globally. After I did this it worked :
```
npm uninstall -g prisma
npm install -g @prisma/cli --save-dev
```
